src/components/select/select.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { type ReactElement, useId } from "react"
+
+import * as SelectPrimitive from "@radix-ui/react-select"
+import clsx from "clsx"
+
+import s from "./select.module.scss"
+
+import { Typography } from "../typography"
+
+type Options = {
+  icon?: string
+  text: string
+  value: string
+}
+
+type Props = {
+  className?: string
+  id?: string
+  labelText?: ReactElement
+  minWidth?: number
+  name: string
+  onValueChange?: (items: string) => void
+  options: Options[]
+  placeholder?: string
+} & React.ComponentPropsWithoutRef<typeof SelectPrimitive.Root>
```

Comment:
React. can be omitted:
`} & ComponentPropsWithoutRef<typeof SelectPrimitive.Root>`